src/xunit.performance.api/XunitRunner.cs

```diff
@@ -4,108 +4,95 @@
 
 using System;
 using System.Threading;
-using System.Reflection;
 using Xunit.Runners;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
 
 namespace Microsoft.Xunit.Performance.Api
 {
-    public class XunitRunner
+    public static class XunitRunner
```

Comment:
I agree with your vision. As runner is just a wrapper on top of xunit, I felt that it was enough being static plus you can still have a list of callbacks (in this case) running sequentially and potentially in parallel (e.g. tests running in different cores to maximize total throughput). These changes reduce the chances to have threading problems in the future because of shared states.

---
In reply to: [100387389](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100387389) [](ancestors = 100387389)